test/e2e/network/conntrack.go

```diff
@@ -143,7 +143,10 @@ var _ = common.SIGDescribe("Conntrack", func() {
 		// Create a pod in one node to create the UDP traffic against the NodePort service every 5 seconds
 		ginkgo.By("creating a client pod for probing the service " + serviceName)
 		clientPod := e2epod.NewAgnhostPod(ns, podClient, nil, nil, nil)
-		clientPod.Spec.NodeName = clientNodeInfo.name
+		nodeSelection := e2epod.NodeSelection{}
+		e2epod.SetAffinity(&nodeSelection, clientNodeInfo.name)
```

Comment:
nit: wouldn't 
```
nodeSelection := e2epod.NodeSelection{Name: clientNodeInfo.name}
```

be easier to follow?

Reply:
you mean removing 
```
		e2epod.SetAffinity(&nodeSelection, clientNodeInfo.name)
```
so we'll have 
```
nodeSelection := e2epod.NodeSelection{Name: clientNodeInfo.name}
e2epod.SetNodeSelection(&clientPod.Spec, nodeSelection)
```
https://github.com/kubernetes/kubernetes/blob/dd6ee99f62ebd91cf22c03a7f2c2594c1f5807e6/test/e2e/framework/pod/node_selection.go#L91-L105

Reply:
yes - that's what I meant